### DISCUSSION TITLE
Circular genome but different when visualised in bandage

### DISCUSSION THREAD

**Discussion by mariaMS23** [Kinggerm/GetOrganelle#277] [General]
Hi Kinggerm,

I assembled chloroplast genome and successfully get the circular genome. However when i visualized in Bandage, it is not complete circular genome. Do I need to make adjustment on the assembly graph? Thank you for you suggestion and help. 
![graph](https://github.com/Kinggerm/GetOrganelle/assets/134999140/6e7094ef-ef42-4065-8271-5c882f97726e)
[get_org.log.txt](https://github.com/Kinggerm/GetOrganelle/files/11597441/get_org.log.txt)

**Comment by JianjunJin** [comment]
You must be visualizing the wrong file or setting some threshold (e.g. depth range) in Bandage.

Nothing else you need to do. The `repeat_pattern1` files are the file complete two results (see explanation at FAQ).
